test-plan.md

```diff
@@ -0,0 +1,100 @@
+# Validating C# Extension for VS Code
+
+#### Opening projects
+When you open a directory in VS Code, the C# extension should look for a .csproj or .sln file in that directory and use "OmniSharp" to load it. 
+If you look in "Output > Omnisharp Log" a bunch of information should be printed about what copy of MSBuild was used and what projects were load
+
+Project types to test:
+* Standalone csproj 
+* Directory containing .sln file that references csprojs--projects should be loaded 
+* .NET Core/.NET Standard csproj
+* (Windows) Desktop .NET projects
+* Unity projects
+
+The easist way to verify that a project was successfully loaded is to open a .cs file within it and verify that the references codelens indicator appears.
+
+#### Specific projects to test opening (on all OSes):
+* `dotnet new console` in a directory
+* A more complex dotnet solution, eg. [1] 
+* A console app solution created in Visual Studio
+
+#### Unity
+* Follow the directions at https://code.visualstudio.com/docs/other/unity to configure VS Code with unity
+* Try editing a project like https://github.com/staceyhaffner/SuperSpaceShooter
+
+#### Intellisense 
+* The completion list in a file contains symbols defined in references and in the file
+* If you write a documentation comment on a symbol, the completion list displays it
+
+#### Signature Help 
+* Signature Help shows up in a method call after typing `(`
+* Signature help shows documentation for methods and for parameters
+
```

Comment:
Maybe we can add something related to the active parameter here, like "Signature Help shows the documentation for the parameter that is currently active"

Reply:
Done